plugins/search-backend-node/src/IndexBuilder.ts

```diff
@@ -37,15 +38,25 @@ type IndexBuilderOptions = {
   logger: Logger;
 };
 
+export type DocumentTypeInfo = {
```

Comment:
Candidate to throw in `@backstage/search-common`?  We don't have a use for this now, but we've thought about exposing a metadata endpoint in the search backend service that returns metadata for types (e.g. what types are there? what are their schemas?) so that the a search page on the front-end could be composed a little more dynamically.

Reply:
Makes sense to me! Done in b97c46a780.